cryodrgn/mrc.py

```diff
@@ -100,14 +104,30 @@ def parse(cls, fname):
 
     @classmethod
     def make_default_header(
-        cls, data, is_vol=True, Apix=1.0, xorg=0.0, yorg=0.0, zorg=0.0
+        cls,
+        nz=None,
+        ny=None,
+        nx=None,
+        data=None,
+        is_vol=True,
+        Apix=1.0,
+        xorg=0.0,
+        yorg=0.0,
+        zorg=0.0,
```

Comment:
I'm pretty sure by doing this you've broken all the call sites that call this with positional args!

(In general it's a bad idea to call a function like this with positional args anyway, I like the `def foo(*, ...)` syntax that prevents that...)

```suggestion
        cls,
        data=None,
        is_vol=True,
        Apix=1.0,
        xorg=0.0,
        yorg=0.0,
        zorg=0.0,
        nz=None,
        ny=None,
        nx=None,
```

Reply:
True - I _would_ have broken client code, but in this case I've modified all the calling code accordingly as well. I do think the newly introduced `nz/ny/nx` belong in the front because they're important. If this function was an API function, I would have added these at the back, but I seriously doubt any `cryoDRGN` users are using this in their code.

Reply:
How about you use the `*` syntax and have all callers use keyword args?